selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/Workspaces.java

```diff
@@ -101,7 +102,7 @@ public String getWorkspaceStatus(String workspaceName) {
   }
 
   public void waitWorkspaceStatusIs(String workspaceName, String workspaceStatus) {
```

Comment:
Ended "Is" looks strange at the end of method name. Could you, please, remove it?